policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/child_assistance/age_eligibility.yaml

```diff
@@ -0,0 +1,11 @@
+description: Children eligible for the Quebec Child Assistance Credit.
```

Comment:
Change description to:
Quebec provides the child assistance tax credit amount for filer's children of or below this age threshold.